docs/zero_to_hero_guide/00_Inference101.ipynb

```diff
@@ -0,0 +1,247 @@
+{
+ "cells": [
```

Comment:
Lets add a note to make sure user know having spawned a server is assumed.

Reply:
I've added a note clarifying that having a server spawned is assumed, and also linked to the get started guide.